document-portal/document-portal-fuse.c

```diff
@@ -3157,10 +3157,16 @@ xdp_fuse_setlk (fuse_req_t req,
                 int sleep)
 {
   const char *op = "SETLK";
+  XdpFile *file = (XdpFile *)fi->fh;
+  int res;
 
   g_debug ("SETLK %lx", ino);
 
-  xdp_reply_err (op, req, ENOSYS);
+  res = fcntl (file->fd, F_SETLK, lock);
```

Comment:
Is it necessary to also implement `xdp_fuse_getlk`?

Reply:
this is done now.